packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -181,12 +169,40 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
   return self;
 }
 
+- (AVCaptureConnection *)createConnection:(NSError **)error {
+  // setup input
```

Comment:
Style nit: In the ObjC style guide, comments should complete sentences, with capitalization and punctuation. This should be fixed throughout the PR.